ipaserver/install/ipa_cacert_manage.py

```diff
@@ -145,6 +148,8 @@ def run(self):
                 return self.list()
             elif command == 'delete':
                 return self.delete()
+            elif command == 'prune':
+                return self.prune()
```

Comment:
I wonder if we have enough commands that than passing we do:
```
if command not in ('renew', 'install', ...):
    parser.error("unknown command \"%s\"" % command)
```

and then do the additional argument enforcing for 'install' and 'delete'.

Reply:
OK, I've just updated that. We could also remove all the if's with function calls by using `getattr(self, command)` but I'm not sure if that is safe enough.